packages/elements/src/components/ui/value-table/ValueTable.module.css

```diff
@@ -2,22 +2,29 @@
   /**
      * Theme vars
      */
-  --swui-value-table-height: 40px;
   width: 100%;
-
   .row:not(:first-child) td {
     border-top: 1px solid var(--lhds-color-ui-300);
   }
+  td {
+    --current-spacing: 1.5;
+    --current-indent: 0;
+    padding: calc(var(--current-spacing) * var(--swui-metrics-spacing))
+      calc(var(--current-indent) * var(--swui-metrics-indent));
+  }
+  tbody {
+    vertical-align: top;
+  }
 }
 
-.relaxed {
-  --swui-value-table-height: 48px;
+.relaxed td {
+  --current-spacing: 2;
```

Comment:
Is this really correct? 
The height should be exactly 48px, which is clearly communicated in the `--swui-value-table-height: 48px;`

Reply:
Yes, we did a SmartThing™ and made the alignment for a label to always be `top` even if the value height grows!

Reply:
as Johan said :)